crates/miden-objects/src/block/mod.rs

```diff
@@ -18,7 +18,7 @@ pub use partial_account_tree::PartialAccountTree;
 
 pub mod account_tree;
 
-mod nullifier_tree;
+pub mod nullifier_tree;
 pub use nullifier_tree::NullifierTree;
```

Comment:
Since we are making the `nullifier_tree` module public, I'd remove the `NullifierTree` re-export (i.e., delete line 22). This would also be consistent with how we export the `account_tree` module.